src/syslog_mcp/services/alert_monitor.py

```
@@ -0,0 +1,357 @@
+        """Async context manager exit."""
+        await self.es_client.disconnect()
+    
+    def load_alert_rules(self) -> List[Dict[str, Any]]:
+        """Load alert rules from storage."""
+        rules_file = Path.home() / ".syslog-mcp" / "alert_rules.json"
+        
+        if not rules_file.exists():
+            return []
+        
+        try:
+            with open(rules_file, 'r') as f:
```

Ruff (0.12.2)

`open()` should be replaced by `Path.open()`

(PTH123)